helm/values-uat.yaml

```diff
@@ -21,6 +21,10 @@ microservice-chart:
       REDIS_PORT: "6380"
       REDIS_SSL_ENABLED: "true"
       CARTS_MAX_ALLOWED_PAYMENT_NOTICES: "5"
+      ELASTIC_APM_SERVICE_NAME: pagopa-ecommerce-payment-requests-service
+      ELASTIC_APM_APPLICATION_PACKAGES: it.pagopa.ecommerce.payment.requests
+      ELASTIC_APM_SERVER_URL: http://quickstart-apm-http.elastic-system.svc:8200
+      ELASTIC_APM_LOG_SENDING: "true"
     envSecret:
       REDIS_PASSWORD: redis-ecommerce-password
```

Comment:
Missing env secret ELASTIC_APM_SECRET_TOKEN (add it after azure update)

Reply:
Correct, thanks :) done with 032f2e7